### PR TITLE
Trying to get vibration when failed fp recognition

### PR DESCRIPTION
**Pull request by valascus**

### CODE REVIEW

File: MediaTek/Fingerprint/fp/src/com/silead/fp/lockscreen/FpService.java [hunk 1]
```diff
@@ -265,7 +266,6 @@ private void onIdentifyError() {
             return;
             } else {
             //Log.d(TAG,"onIdentifyError");
-            Vibrator vibAttempt = (Vibrator) this.context.getSystemService(Context.VIBRATOR_SERVICE);
             vibAttempt(1000);
```
Comment (janengelmohr) [inline MediaTek/Fingerprint/fp/src/com/silead/fp/lockscreen/FpService.java:269]: This throws an error now because vibAttempt is named v now ;)

**Comment by valascus** [comment]
I didnt want to make a pull request. i was changing mine, but the github software is a bit confuse xD
sorry.

**Comment by janengelmohr** [comment]
No problemo :)

**Comment by janengelmohr** [comment]
I am also experimenting with this feature now - good idea!

**Pull request closed without merging**